jupyterbook/environment.yml

```diff
@@ -0,0 +1,82 @@
+name: IOOS
```

Comment:
Can we read from the .binder directory instead? To avoid duplication of this file.

Reply:
Sure. We don't actually need the env.yml file for building the web app. Will remove it.